docs/scheduler/filter.md

```diff
@@ -11,22 +11,99 @@ weight=4
 
 # Filters
 
-The `Docker Swarm` scheduler comes with multiple filters.
+Filters tell Docker Swarm which nodes to use when creating and running a
```

Comment:
Just an idea, but we can probably be more explicit here:

```
Filters tell Docker Swarm the subset of nodes to keep when creating and running a container. 
It filters out nodes that don't fall into certain categories in the process of scheduling 
the container on the cluster. It is the first step of the scheduling workflow. The strategy
then picks the right node from the subset of nodes we get after applying the filters.
```

Not sure if more clear, but we can be more explicit as to what filters are and what are their purpose in the `filter->strategy` flow.